src/openrct2/ride/TrackData.cpp

```diff
@@ -6308,6 +6317,7 @@ static constexpr uint32_t TrackFlags[] = {
     /* TrackElemType::RightEighthBankToOrthogonalDown25      */   TRACK_ELEM_FLAG_TURN_RIGHT | TRACK_ELEM_FLAG_TURN_SLOPED | TRACK_ELEM_FLAG_DOWN | TRACK_ELEM_FLAG_STARTS_AT_HALF_HEIGHT | TRACK_ELEM_FLAG_ALLOW_LIFT_HILL | TRACK_ELEM_FLAG_CURVE_ALLOWS_LIFT,  
     /* TrackElemType::DiagBrakes                                    */   0,
     /* TrackElemType::DiagBlockBrakes                               */   0,
+    /* TrackElemType::Down25                                        */   TRACK_ELEM_FLAG_DOWN | TRACK_ELEM_FLAG_STARTS_AT_HALF_HEIGHT,
```

Comment:
/* TrackElemType::SlopedBrakes

Reply:
You must have seen an outdated version; this comment has been corrected.